torchtune/models/llama3_2_vision/_model_builders.py

```diff
@@ -167,6 +167,11 @@ def lora_llama3_2_vision_11b(
     decoder_type = LoRATrainable(decoder_trainable.lower())
     encoder_type = LoRATrainable(encoder_trainable.lower())
     fusion_type = LoRATrainable(fusion_trainable.lower())
+    assert LoRATrainable.FULL not in [
+        decoder_type,
+        decoder_type,
+        fusion_type,
+    ], "We've temporarily removed support for mixed LoRA + Full Finetuning yet. Please don't use the 'full' option and use llama3_2_vision_11b if you need full finetuning"
```

Comment:
I would just say "X is not currently supported" instead of "We've temporarily removed support for X" (and same comment on the error message further down)

Reply:
I worded it this way since the option is still there, but we don't let you use it. Also in case someone is already using it.

Reply:
> We've temporarily removed support for mixed LoRA + Full Finetuning yet.

😕

Reply:
Yeah this could be worded in a kinder manner. Feels aggressive. Ask ChatGPT to reword or smthing.